docs/guide/stories.md

```diff
@@ -63,7 +63,7 @@ Components are rendered in the default application layout. The layout for a set
 ```ruby
 # test/components/stories/header_componeont_stories.rb
 class HeaderComponentStories < ViewComponent::Storybook::Stories
-  layout :desktop
+  layout 'desktop'
```

Comment:
Symbol layouts are fine. What drives this change?

Reply:
Hmm. I still can't make this work with a symbol layout. The `StoriesController` calls Rails' [`render`](https://api.rubyonrails.org/v6.1.4/classes/AbstractController/Rendering.html#method-i-render) method which doesn't allow symbols. Maybe I'm doing something wrong?

https://github.com/jonspalmer/view_component_storybook/blob/main/app/controllers/view_component/storybook/stories_controller.rb#L23

<img width="1040" alt="Screenshot 2021-11-30 at 11 39 21 AM" src="https://user-images.githubusercontent.com/5836587/143955671-088272f4-516a-46e7-8a6f-b8fb86691e28.png">

Reply:
Just tested out both strings and symbols in my project @developius - which version of Rails are you using? (I doubt this is the issue based on your screen shot but it's my only thought at the moment).

Reply:
@hjhart woah strange. This is a Rails 6.1.4.1 project with Ruby 3.0.2. I'll take a deeper look into this tomorrow and see if there's something out of whack.

Reply:
If it helps: 

```
module Prism
  class AlertStories < ViewComponent::Storybook::Stories
    layout :component_preview

    story(:default) do
      constructor(
        message: "Please pay attention to this alert because it's important"
      )
    end
  end
end

Reply:
@hjhart Happy New Year 😆 still running into this error when using a symbol, any thoughts? Happy to remove this change from the PR but wondering if anyone else is running into it too.